docs/UserGuide.md

```diff
@@ -81,6 +83,16 @@ e.g. `07:00` is a valid time input and refers to 7am, but not `7:00`
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </box>
 
+** Notes on Fields for the students **
+
+| Field       | Prefix    | Required | Caveats                                                                                                                                                                                                                                                                                                                                                                               |
+|-------------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Name**    | `-name`   | Yes      | Names should only contain alphanumeric characters and spaces, it cannot contain special character, do note that long names might cause the Id field to be blocked, in this case you are able to drag the middle portion of the UI to reveal the ID, the upper limit of this workaround would be if the name is 220 characters long. In this case do proceed to use nicknames instead. |
```

Comment:
Some grammar/punctuation fixes:

`Names should only contain alphanumeric characters and spaces, it cannot contain special characters. Long names might also cause the Id field to be blocked. In this case you are able to drag the middle portion of the UI to reveal the ID. However the upper limit of this workaround would be 220 characters long. If a student's name exceeds 220 characters, it is advisable to use nicknames instead.`